### PR TITLE
use static fixtures for registers lookup

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/Hey9OS9t/805-2-mitigate-registers-switch-off

In order to mitigate registers switch off, we can store the data as static fixtures to maintain the existing interface.

### CODE REVIEW

File: dmscripts/helpers/supplier_data_helpers.py [hunk 1]
```diff
@@ -329,17 +326,25 @@ def get_supplier_ids_from_args(args):
     return supplier_ids
 
 
-@lru_cache()
```
Comment (bjgill) [inline dmscripts/helpers/supplier_data_helpers.py:332]: Why are we removing the cache?

Comment (gidsg) [reply dmscripts/helpers/supplier_data_helpers.py:332]: Ah, good spot 👀 
I removed this as I assumed it was something to do with caching the HTTP response but now I see it is a more generic thing, I have restored it now in 7e05fc0

**Review by bjgill — APPROVED** [review]
Changes all look sensible.

Possibly worth recording how you generated the two JSON files? I assume you just downloaded them from Registers today?

**Comment by gidsg** [comment]
> I assume you just downloaded them from Registers today?

@bjgill yes, literally that, I can denote it in an extended commit message as JSON doesn't support comments.

**Pull request merged**